### ISSUE TITLE
Data Call: Group Members

### ISSUE THREAD

**Issue by lachie-white** [terraform-providers/terraform-provider-azuread#191]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Enable the ability to call AAD and retrieve a list of users from within the group.
This enables for things like iterating through creation of KV access policies via AG Groups

### New or Affected Resource(s)

* azuread_group_members

### Potential Terraform Configuration

```hcl
data "azurerad_group_members" "members" {
    name = "aad_test_group"
}
```

### References

* #0000

**Comment by dekimsey** [comment]
I found this is already implemented, it's just undocumented.

https://github.com/terraform-providers/terraform-provider-azuread/blob/master/azuread/data_group.go#L38

**Comment by katbyte** [comment]
Hi @lachie-white, this was fixed by #177 🙂 as such closing
